files/default/templates/hopsworks.email.tmpl

```diff
@@ -0,0 +1,420 @@
+                {{ end }}
+                {{ range .Alerts.Firing }}
+                <tr>
+                  <td class="content-block">
+                    {{ if gt (len .Annotations) 0 }}
+                      <strong>Alert:</strong> {{ .Annotations.title }} {{ if .Labels.severity }} - {{ .Labels.severity }} {{ end }}<br />
+                      <strong>Summary:</strong> {{ .Annotations.summary }} <br />
+                      <strong>Description:</strong> {{ .Annotations.description }} <br />
+                    {{ end }}
+                    <strong>Details:</strong><br />
+                    <ul>
+                      <li> Project: {{ .Labels.project }} </li>
+                      {{ if .Labels.job -}} <li> Job: {{ .Labels.job }} </li> {{ end }}
+                      {{ if .Labels.executionId -}} <li>  Execution id: {{ .Labels.executionId }} </li> {{ end }}
+                      {{ if .Labels.featureGroup -}} <li>  Feature Group: {{ .Labels.featureGroup }} </li> {{ end }}
+                      {{ if .Labels.featureGroupId -}} <li>  Feature Group id: {{ .Labels.featureGroupId }} </li> {{ end }}
+                      {{ if .Labels.status -}} <li>  Status: {{ .Labels.status }} </li> {{ end }}
+                    </ul>
+                  </td>
+                </tr>
+                {{ end }}
+
+                {{ if gt (len .Alerts.Resolved) 0 }}
+                  {{ if gt (len .Alerts.Firing) 0 }}
+                <tr>
+                  <td class="content-block">
+                    <br />
+                    <hr />
+                    <br />
+                  </td>
+                </tr>
+                  {{ end }}
+                <tr>
+                  <td class="content-block">
+                    <strong>[{{ .Alerts.Resolved | len }}] Resolved</strong>
+                  </td>
+                </tr>
+                {{ end }}
+                {{ range .Alerts.Resolved }}
+                <tr>
+                  <td class="content-block">
+                    {{ if gt (len .Annotations) 0 }}
+                      <strong>Alert:</strong> {{ .Annotations.title }} {{ if .Labels.severity }} - {{ .Labels.severity }} {{ end }}<br />
+                      <strong>Summary:</strong> {{ .Annotations.summary }} <br />
+                      <strong>Description:</strong> {{ .Annotations.description }} <br />
+                    {{ end }}
+                    <strong>Details:</strong><br />
+                    <ul>
+                      <li> Project: {{ .Labels.project }} </li>
+                      {{ if .Labels.job -}} <li> Job: {{ .Labels.job }} </li> {{ end }}
+                      {{ if .Labels.executionId -}} <li>  Execution id: {{ .Labels.executionId }} </li> {{ end }}
+                      {{ if .Labels.featureGroup -}} <li>  Feature Group: {{ .Labels.featureGroup }} </li> {{ end }}
+                      {{ if .Labels.featureGroupId -}} <li>  Feature Group id: {{ .Labels.featureGroupId }} </li> {{ end }}
```

Comment:
Feature group id is useless. The alert should contain feature store name/ feature group name/feature group version.